cmd/cometbft/commands/reset.go

```diff
@@ -18,16 +18,14 @@ var ResetAllCmd = &cobra.Command{
 	Aliases: []string{"unsafe_reset_all"},
 	Short:   "(unsafe) Remove all the data and WAL, reset this node's validator to genesis state",
 	RunE:    resetAllCmd,
-	PreRun:  deprecateSnakeCase,
 }
 
 var keepAddrBook bool
 
 // ResetStateCmd removes the database of the specified CometBFT core instance.
 var ResetStateCmd = &cobra.Command{
-	Use:    "reset-state",
-	Short:  "Remove all the data and WAL",
-	PreRun: deprecateSnakeCase,
+	Use:   "reset-state",
```

Comment:
FYI most other commands have a snake_case alias (eg `gen_validator`, `probe_upnp`..) but `reset-state` does not have.